rust-toolchain

```diff
@@ -1 +1,5 @@
-1.63
+# rust-toolchain.toml
+[toolchain]
+channel = "1.63"
+components = [ "rustfmt", "clippy" ]
+targets = [ "wasm32-unknown-unknown", "wasm32-wasi" ]
```

Comment:
Is that change necessary? why imposing wasm32 targets or rustfmt/clippy for people that just plan on building wasmer?

Reply:
Yeah it's a leftover from when I tested cargo wapm

Reply:
ok done

Reply:
turns out it was necessary to run the integration tests. wasm32-wasi is not a very large download, so I'll just use `targets = ["wasm32-wasi"]`